charts/pg-db/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: pg-db
 description: 'A Helm chart to deploy the PostgreSQL database with the Percona Operator for PostgreSQL'
 type: application
-version: 2.2.2
+version: 2.2.4
```

Comment:
Why increase two versions?

Reply:
we have 2.3.0 in 
https://github.com/percona/percona-helm-charts/pull/232/files#diff-64472c1e36d35d9916e54a8a05dbc0b508cb4268e822ee5ad69846e44f28989c